Using-tsconfig.json-in-Visual-Studio-2015.md

```diff
@@ -0,0 +1,41 @@
+## Overview
+
+[`tsconfig.json`](tsconfig.json.md) has limited support in Visual Studio 2015.
+Current support is limited to 'loose files', ASP.NET v5, and Apache Cordova Apps.
+
+*Note:* At this time we don't support any other project types in Visual Studio using a `tsconfig.json` file.
+This include the 'HTML Application with TypeScript' and ASP.NET v4.
+
+## Loose Files
+
+Any time you open a TypeScript project in Visual Studio, we will do a search for a `tsconfig.json` file by traversing up each
+directory to the root directory, or as soon as we find a `tsconfig.json` file.
```

Comment:
I would re-word 
"... traversing up each directory to the root directory **until** we find ..."